### DISCUSSION TITLE
Tapping Style mod

### DISCUSSION THREAD

**Discussion by Euro20179** [ppy/osu#17772] [Ideas]
It would replace the Alternate mod and contain 2 options, single tap, and alternate. Only 1 can be enabled. Alternate would be enabled by default.
If any other mods that change how you can tap get added, they could possibly also be put here.

Should this be in the game? Thoughts?

I started implementing whether or not it should be in the game anyway cause im bored.

**Comment by respektive** [comment]
I think having a single tap option for alternate mod would be a nice addition. 
Probably makes sense to rename the mod if more tapping styles are added, but I think if it's just the one it should be fine to keep it as alternate. not sure

**Reply by Euro20179** [reply]
In my opinion its a bit weird to have a mod called "alternate" with an option called "single tap". However, I think this makes more sense than having a seperate "single tap" mod, as they're so similar.

**Reply by Euro20179** [reply]
I cant edit my reply so im making another.

It didnt occure to me that, instead of having an option, it could be one of those mods that when you click it twice a different mod happens, eg 4k, 5k, 6k etc.. in mania

**Reply by respektive** [reply]
I think stacked mods like that will probably be gone with the upcoming mod select designs. If they do however stay I think that's a valid option.

**Reply by NotGumballer91** [reply]
> In my opinion its a bit weird to have a mod called "alternate" with an option called "single tap". However, I think this makes more sense than having a separate "single tap" mod, as they're so similar.

Having it separate is unnecessary. And having it customizable is also unnecessary.
It should be swappable mod like, SD and PF (as @Euro20179 said)
